synapse/config/server.py

```diff
@@ -72,6 +72,15 @@ def read_config(self, config):
         # master, potentially causing inconsistency.
         self.enable_media_repo = config.get("enable_media_repo", True)
 
+        # whether to require users to authenticate in order to query /profile
+        # endpoints via CS API. this is a workaround in advance of MSC1301 landing
+        self.auth_profile_reqs = config.get("auth_profile_reqs", False)
```

Comment:
why `reqs` ? we don't use that anywhere else and it honestly took me a while to parse it.

Reply:
feel free to rename to something which better reflects what is going on.